api/src/main/java/com/intendia/openapi/OpenApi.java

```diff
@@ -406,10 +407,19 @@
 
     @JsonInclude(NON_ABSENT) class Header {}
 
-    @JsonInclude(NON_ABSENT) class Security {}
-
-    @JsonInclude(NON_ABSENT) class SecurityRequirement {
-        @Override public String toString() { return MoreObjects.toStringHelper(this).toString(); }
+    //TODO: Add OAuth2 fields
+    @JsonInclude(NON_ABSENT) class SecurityDefinition {
```

Comment:
I can change it, but I went with SecurityDefinition since it is one item in Swagger2's securityDefinitions section.

The other bits in this file, I'm not really in love with, but I wasn't sure how to get nice Java syntax on top of what's an array of maps in the json